Makefile

```diff
@@ -22,6 +24,16 @@ pkg/%.min.js: pkg/%.js
 	@echo Writing $@
 	@$(UGLIFYJS) --preamble "$$(tools/preamble)" $< >$@
 
+# compile Sass
+sass:
```

Comment:
For instance, here, we could list the dependencies after the colon. That way, it will only recompile when necessary.